url.bs

```diff
@@ -2511,8 +2510,8 @@ takes a list of name-value or name-value-type tuples <var>tuples</var>, optional
  <li>Return <var>output</var>.
 </ol>
 
-<p class="note no-backref">The <cite>HTML standard</cite> invokes this algorithm with
-name-value-type tuples. [[HTML]]
+<p class="note no-backref">The <cite>HTML standard</cite> invokes this algorithm with name-value
+tuples. [[HTML]]
```

Comment:
You might want to instead say "The HTML standard invokes this algorithm with values that are files" or something similar.

Reply:
Done.